### PR TITLE
Add missing types

### PR DESCRIPTION
**Pull request by ronal2do**

### CODE REVIEW

File: src/ConnectionFromMongoAggregate.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ async function connectionFromMongoAggregate<LoaderResult, Ctx>({
   // avoid large objects retrieval from collection
   const slice: Array<{ _id: ObjectId }> = await clonedAggregate.project('_id');
 
-  const edges = slice.map((value, index) => ({
+  const edges: Array<Object> = slice.map((value, index) => ({
```
Comment (JCMais) [inline src/ConnectionFromMongoAggregate.js]: Better:
```
const edges: Array<{
    cursor: string,
    node: LoaderResult,
}>
```

File: src/ConnectionFromMongoCursor.js [hunk 2]
```diff
@@ -185,10 +185,10 @@ async function connectionFromMongoCursor<LoaderResult, Ctx>({
   clonedCursor.skip(skip);
   clonedCursor.limit(limit);
 
-  //avoid large objects retrieval from collection
+  // avoid large objects retrieval from collection
   const slice: Array<{ _id: ObjectId }> = await clonedCursor.select({ _id: 1 }).exec();
 
-  const edges = slice.map((value, index) => ({
+  const edges: Array<Object> = slice.map((value, index) => ({
```
Comment (JCMais) [inline src/ConnectionFromMongoCursor.js]: same than above

File: src/MongooseLoader.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ function indexResults(results, indexField, cacheKeyFn = key => key) {
 function normalizeResults(keys, indexField, cacheKeyFn = key => key) {
   return results => {
     const indexedResults = indexResults(results, indexField, cacheKeyFn);
+    // $FlowFixMe
```
Comment (ronal2do) [inline src/MongooseLoader.js]: @JCMais and for this case?

File: src/ConnectionFromMongoAggregate.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ronal2do) [reply src/ConnectionFromMongoAggregate.js]: Should I use the code that @JCMais has commented on?

File: src/MongooseLoader.js (hunk 3 above)
Comment (JCMais) [reply src/MongooseLoader.js]: what is flow saying about this line?

**Pull request merged**